relayer/pkg/chainlink/ocr2/transmissions_cache.go

```diff
@@ -133,9 +134,8 @@ func (c *transmissionsCache) LatestRoundRequested(
 	c.tdLock.RLock()
 	defer c.tdLock.RUnlock()
 	configDigest = c.transmissionDetails.Digest
-	epoch = c.transmissionDetails.Epoch
-	round = c.transmissionDetails.Round
-	err = c.assertTransmissionsNotStale()
```

Comment:
got rid of the error check because we don't actually care what the config digest is and asserting an error would actually cause you to transmit more rounds

Reply:
oh wait, actually the Reader needs to implement it because it must conform to the interface was well

Reply:
i removed the need to query the chain and to use the lock